azure/macos/test.yml

```diff
@@ -0,0 +1,32 @@
+parameters:
+  runTestsName: ''
+  runTestsParameters: ''
+
+steps:
+  - script: |
+      export MYSQL_TEST_USER=root
+      export MYSQL_TEST_PASSWD=
+      export PDO_MYSQL_TEST_DSN="mysql:host=localhost;dbname=test"
+      export PDO_MYSQL_TEST_USER=root
+      export PDO_MYSQL_TEST_PASS=
```

Comment:
If we're not going to run mysql tests on mac, just test the builds, then these and other related mysql stuff can be removed - at a glance, it looks like mysql should be tested ....